backend/composer/api/views.py

```diff
@@ -313,6 +313,19 @@ def get_queryset(self):
             return ConnectivityStatement.objects.excluding_draft()
         return super().get_queryset()
 
+    def update(self, request, *args, **kwargs):
+        # Remove 'state' from the request data
+        data = request.data.copy()
+        data.pop('state', None)
```

Comment:
@afonsobspinto suggestion: would this work: pop it from the request.data and then call `.super()`